holoviews/ipython/__init__.py

```diff
@@ -260,7 +260,7 @@ def load_logo(cls, logo: dict | bool = False, bokeh_logo=False, mpl_logo=False,
             'bokeh_version':  bokeh_version,
             'mpl_version':    mpl_version,
             'plotly_version': plotly_version,
-            'show_tooltip': not os.getenv("HV_DOCS_BUILD")
+            'show_tooltip': not (os.getenv("CI") or os.getenv("HV_DOCS_BUILD"))
```

Comment:
How about something like that? I.e. `HV_HIDE_TOOLTIP_LOGO` takes precedence if set to true or false, then `CI`, and finally it's true by default. The idea being that if someone wants to show the tooltip in their docs they should be able to do it without having to touch every `hv.extension()` call.

```python
if os.getenv("HV_HIDE_TOOLTIP_LOGO") is not None:
    show_tooltip = os.getenv("HV_HIDE_TOOLTIP_LOGO", "").lower() in ("0", "false")
elif os.getenv("CI"):
    show_tooltip = False
else:
    show_tooltip=True
```